packages/logs/src/domain/logsCollection/console/consoleCollection.ts

```diff
@@ -19,24 +19,25 @@ const LogStatusForApi = {
   [ConsoleApiName.warn]: StatusType.warn,
   [ConsoleApiName.error]: StatusType.error,
 }
-export function startConsoleCollection(configuration: LogsConfiguration, sender: Sender) {
+export function startConsoleCollection(configuration: LogsConfiguration, lifeCycle: LifeCycle) {
   const consoleObservable = initConsoleObservable(configuration.forwardConsoleLogs)
   const consoleSubscription = consoleObservable.subscribe(reportConsoleLog)
 
   function reportConsoleLog(log: ConsoleLog) {
-    let messageContext: Partial<LogsEvent> = {}
-    if (log.api === ConsoleApiName.error) {
-      messageContext = {
-        error: {
-          origin: ErrorSource.CONSOLE, // Todo: Remove in the next major release
-          stack: log.stack,
-        },
-      }
-    }
-    if (isExperimentalFeatureEnabled('forward-logs')) {
-      messageContext.origin = ErrorSource.CONSOLE
-    }
-    sender.sendToHttp(log.message, messageContext, LogStatusForApi[log.api])
+    lifeCycle.notify(LifeCycleEventType.RAW_LOG_COLLECTED, {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏we could inline this function to avoid to name it